scripts/zones/Halvung/MobIDs.lua

```diff
@@ -1,6 +1,7 @@
-BIG_BOMB    = 17031401;
-DEXTROSE    = 17031598;
-REACTON     = 17031599;
-ACHAMOTH    = 17031600;
+ACHAMOTH                = 17031600;
+BIG_BOMB                = 17031401;
+DEXTROSE                = 17031598;
+GURFURLUR_THE_MENACING  = 17031592;
```

Comment:
again, pls just add the new one without shuffling the existing lines. They were in order of ID all neat and tidy and now OCD is triggered. That's not a thing I can control, it just is.